src/bun.zig

```diff
@@ -915,6 +915,13 @@ pub const SignalCode = enum(u8) {
         return null;
     }
 
+    pub fn toExitCode(value: SignalCode) ?u8 {
```

Comment:
please add a comment saying that this is only to be used in Shell scripting, where this is the observed behavior of signals. interpreting a signal code as an exit code is almost always incorrect.